app/assets/stylesheets/syntax.scss

```diff
@@ -13,214 +13,220 @@ div.inner-comment div.body div.highlight pre.highlight {
   }
 }
 
+$code-highlight-background-color: #29292e;
+$code-highlight-text-color: #89c4f4;
+$code-highlight-comment-color: #d5d5d5;
+$code-highlight-declaration-color: #f39c12;
+$code-highlight-literal-color: #dda0dd;
+$code-highlight-error-color: #f9690e;
+$code-highlight-name-color: #7ed07e;
+$code-highlight-string-color: #f2ca27;
+
 .highlight .hll {
   background-color: #49483e;
 }
 .highlight {
-  background: #29292e;
-  color: #f8f8f2;
+  background: $code-highlight-background-color;
+  color: $code-highlight-text-color;
 }
 .highlight .c {
-  color: #75715e;
+  color: $code-highlight-comment-color;
 } /* Comment */
-.highlight .err {
```

Comment:
Any reason for removing `.err` styling?

Reply:
@ludwiczakpawel Yes, it's not a good idea to remove error syntax highlighting. I added it back with an updated definition, as the original one was redundant and with same colors it always applies the biggest radius. I made it a little bit smaller as 10px makes it too visible imho.